gluten-ut/spark34/src/test/scala/org/apache/gluten/utils/velox/VeloxTestSettings.scala

```diff
@@ -750,6 +750,8 @@ class VeloxTestSettings extends BackendTestSettings {
   enableSuite[GlutenFileFormatWriterSuite]
   enableSuite[GlutenFileIndexSuite]
   enableSuite[GlutenFileMetadataStructSuite]
+  enableSuite[GlutenFileMetadataStructRowIndexSuite]
+    .exclude("reading _tmp_metadata_row_index - present in a table")
```

Comment:
Seems you rewrite this test? please add comments before line.

Reply:
sure, will do